src/NodaTime/Instant.cs

```diff
@@ -533,11 +534,20 @@ public static Instant Max(Instant x, Instant y)
             BclEpoch.PlusTicks(dateTimeOffset.Ticks - dateTimeOffset.Offset.Ticks);
 
         /// <summary>
-        /// Converts a Julian Day Number into a new Instant representing the same instant in time
+        /// Converts a Julian Date (JD) into a new <see cref="Instant"/> representing the given number of days
```

Comment:
Arguably it's the Julian Date that represents the given number of days, not the Instant.

Perhaps "Converts a Julian Date represented as the given number of days since [...] into a new Instant"?